Source/WebCore/css/CSSSelectorList.cpp

```diff
@@ -175,7 +175,7 @@ class SelectorHasInvalidSelectorFunctor {
 public:
     bool operator()(const CSSSelector* selector)
     {
-        return selector->isUnknownPseudoElement() || selector->isCustomPseudoElement();
```

Comment:
That would be a good opportunity to convert the whole functor class to a simple lambda.